### PR TITLE
Only include symbols / src in transport packages, also fix pre-existing typo

### PR DESCRIPTION
**Pull request by chcosta**
https://github.com/dotnet/corefx/issues/17256

Only includes src / symbols in the transport packages.  Also, fixes a long-standing (unrelated) typo that I noticed in dir.props.

@ericstj @weshaggard

### CODE REVIEW

File: pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.pkgproj [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <HarvestStablePackage>false</HarvestStablePackage>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <PackageIndexFileName>packageIndex.json</PackageIndexFileName>
+
+    <!-- Include symbols in package by default-->
+    <IncludeSymbolsInPackage Condition="'$(IncludeSymbolsInPackage)' == ''">true</IncludeSymbolsInPackage>
```
Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.pkgproj]: NIT: For this one it's not required.  This is just a data package with some targets and a json file.

File: dir.props [hunk 2]
```diff
@@ -187,7 +187,7 @@
     <NuGetRuntimeIdentifier Condition="'$(TargetGroup)' == 'netcore50aot'">win8-aot</NuGetRuntimeIdentifier>
 
     <!-- workaround Dev14 issue with nuget targets -->
-    <RuntimeIndentifier>$(NuGetRuntimeIdentifier)</RuntimeIndentifier>
+    <RuntimeIdentifier>$(NuGetRuntimeIdentifier)</RuntimeIdentifier>
```
Comment (ericstj) [inline dir.props]: This misspelling is intentional.  The Dev14 RTM targets had the same misspelling.

Comment (chcosta) [inline dir.props]: wow, that's sad.  ok

Comment (ericstj) [inline dir.props]: It was fixed in update1 I think.  Perhaps we can remove it as some point.

**Comment by karelz** [comment]
@chcosta does it fix #17256? If yes, can you change the to post to "Fixes #17256" to kick off auto-close upon merge?

**Pull request merged**